src/dags/common/__init__.py

```diff
@@ -34,10 +38,26 @@ def slack_failed_task(context):
     return failed_alert.execute(context=context)
 
 
+# set the local time zone, so the start_date DAG param can use it in its context
+local_tz = pendulum.timezone("Europe/Amsterdam")
+
+# set start_date 'yesterday', so it can be used to seperate integer values for YYYY MM DD
+start_date_dag = str(days_ago(1))
+YYYY = 0
+MM = 0
+DD = 0
+
+# extract the YYYY MM and DD values as integers
+get_YYYY_MM_DD_values = re.search("([0-9]{4})-([0-9]{2})-([0-9]{2})", start_date_dag)
+if get_YYYY_MM_DD_values:
+    YYYY = int(get_YYYY_MM_DD_values.group(1))
+    MM = int(get_YYYY_MM_DD_values.group(2))
+    DD = int(get_YYYY_MM_DD_values.group(3))
+
 default_args = {
     "owner": "dataservices",
     "depends_on_past": False,
-    "start_date": days_ago(1),
+    "start_date": datetime(YYYY, MM, DD, 2, tzinfo=local_tz),
```

Comment:
Ik neem aan dat de bedoeling is om 2 uur s'nachts Amsterdam tijdzone

Reply:
Ik denk dat het cijfer 2 weg kan, is bedoeld om het verschil aan te duiden tussen UTC en Amsterdam. Ga ik nog testen